icu4c/source/i18n/unicode/udatpg.h

```diff
@@ -651,4 +669,14 @@ udatpg_getPatternForSkeleton(const UDateTimePatternGenerator *dtpg,
                              const UChar *skeleton, int32_t skeletonLength,
                              int32_t *pLength);
 
+/**
+ * Return the default hour cycle.
+ *
+ * @param dtpg a pointer to UDateTimePatternGenerator.
+ * @return the default hour cycle.
+ * @stable ICU 67
+ */
+U_DRAFT UDateTimeHourCycle U_EXPORT2
+udatpg_getDefaultHourCycle(const UDateTimePatternGenerator *dtpg);
```

Comment:
I'd make this take an error code, even if you don't need to use it right now.  If the code backing this API were refactored in the future in a way that would need the error code, it's good to have it.

Reply:
The API proposal was approved this morning without the error code.

@FrankYFTang Can you send an email to icu-core asking if we should add the error code?

Reply:
sorry, I forgot about that.